build/shade/_k-standard-goals.shade

```diff
@@ -189,12 +189,15 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
 
   copy sourceDir='${Path.GetDirectoryName(project)}' include='bin/**/' outputDir='${Path.Combine(BUILD_DIR, Path.GetFileNameWithoutExtension(project))}' overwrite='${true}' each='var project in nativeProjects'
 
-#nuget-verify target='package' if='File.Exists(PACKAGELIST_JSON_FILENAME) && ShouldVerifyNupkgs' description='Verify if all the packages are generated properly'
+#nuget-verify target='package' description='Verify if all the packages are generated properly'
```

Comment:
This allows us to invoke the target but no-op it. Having the `if` in the declaration, makes the target not be declared.